climada/entity/exposures/base.py

```diff
@@ -906,15 +906,29 @@ def copy(self, deep=True):
             **metadata
         )
 
-    def write_raster(self, file_name, value_name='value', scheduler=None):
+    def write_raster(self, file_name, value_name=['value'], scheduler=None):
         """Write value data into raster file with GeoTiff format
 
         Parameters
         ----------
         file_name : str
-            name output file in tif format
+            output tif file to write to
+        value_name : str or list of str
+            column(s) in the geodataframe containing data. If multiple values are provided, each column will be
+            written to a separate band in the output raster
+        scheduler : str, optional
+            used for dask map_partitions.
+            “threads”, “synchronous” or “processes”
         """
-        if self.meta and self.meta['height'] * self.meta['width'] == len(self.gdf):
+        if not isinstance(value_name, list):
+            value_name = [value_name]
+
+        # Simple case: one raster band, every grid value defined in the Exposures.
+        # Then we can skip a slow points_to_raster call:
+        if self.meta and \
+                'height' in self.meta.keys() and \
+                len(value_name) == 1 and \
+                self.meta['height'] * self.meta['width'] == len(self.gdf):
```

Comment:
Will this not cause an error if `meta['height']` or `meta['width']` are not defined? 

Also, the clause `'height' in self.meta.keys()` is probably redundant. OR, it should read:

```suggestion
        if self.meta and 'height' in self.meta and 'width' in self.meta:
                if len(value_name) == 1 and self.meta['height'] * self.meta['width'] == len(self.gdf):
```

Note that checking if something is in a dict by default looks at the keys.

Reply:
Would you go for the nested `if`s over the awkward indentation?
Thanks for the tip about dictionary keys.

Reply:
Yes, it makes the code much clearer and safer.

Reply:
This should eventually be replaced by the `is_meta` method.